### ISSUE TITLE
Selector and refiner training issue

### ISSUE THREAD

**Issue by EvdoTheo** [liuyuan-pal/Gen6D#120]
Hello @liuyuan-pal, I'm trying to train the selector on my custom data and the loss score is not decreasing, instead, it is stable at 0.4. What do you suggest in order to unstuck it?

Moreover, I also try to train the refiner, but the new refiner is giving me worse results than the pretrained one. For example, when the refiner starts working after a few frames it completely loses its target and the detection algorithm must be deployed to initialize pose. I attached the inter folder to give it a glance. Thank you!

https://drive.google.com/drive/folders/157W5knfP3X2OPWmrwg9QuDpiHvNLgpq7?usp=sharing

**Comment by liuyuan-pal** [comment]
Hi, I'm not sure about the problem here. We may need to visualize the intermediate results and check the model is able to overfit a small training set for sanity check purposes.
